vista_3d/README.md

```diff
@@ -4,15 +4,7 @@
 The codebase is under Apache 2.0 Licence. The model weight is released under [NVIDIA OneWay Noncommercial License](./NVIDIA%20OneWay%20Noncommercial%20License.txt).
 
 ## Reference
-
-```
-@article{he2024vista3d,
-  title={VISTA3D: Versatile Imaging SegmenTation and Annotation model for 3D Computed Tomography},
-  author={He, Yufan and Guo, Pengfei and Tang, Yucheng and Myronenko, Andriy and Nath, Vishwesh and Xu, Ziyue and Yang, Dong and Zhao, Can and Simon, Benjamin and Belue, Mason and others},
-  journal={arXiv preprint arXiv:2406.05285},
-  year={2024}
-}
-```
+[1] Yufan He, Pengfei Guo, Yucheng Tang, Andriy Myronenko, Vishwesh Nath, Ziyue Xu, Dong Yang, Can Zhao, Benjamin Simon, Mason Belue, Stephanie Harmon, Baris Turkbey, Daguang Xu and Wenqi Li: "VISTA3D: Versatile Imaging SegmenTation and Annotation model for 3D Computed Tomography". (2024), [arXiv](https://arxiv.org/abs/2406.05285)
```

Comment:
Why do we need to change the format of the citation here? The original is bibtex and easier to use.

Reply:
Yes, I was also a bit concerned because QA wanted a link. However, I reverted that change because having the arXiv ID also makes it easy to find the link.